boards/x86/qemu_x86/qemu_x86_64_coverage.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: qemu_x86_64_coverage
```

Comment:
why i sthis  platform definition needed?

Reply:
Because there was already qemu_x86_64 identifier exist, that can let us decide whether running with coverage report or not. I try to mimic similar configurations from qemu_x86 and qemu_x86_coverage.  
we can run command as follow:
west build tests/kernel/ - b qemu_x86_64_coverage    or    sanitycheck -T tests/kernel/ -b qemu_x86_64  --coverage
thanks!

Reply:
qemu_x86_coverage is there for different reasons, we do not need to add a special coverage platform for each platform we enable coverage

Reply:
Ok, understand. I'll see other example and correct this, thanks!

Reply:
I already update this, could you please help to review it again, @nashif  thanks!